packages/react-composites/tests/browser/call/hermetic/HorizontalGallery.test.ts

```diff
@@ -73,4 +74,117 @@ test.describe('HorizontalGallery tests', async () => {
       'horizontal-gallery-with-many-audio-participants-on-page-1.png'
     );
   });
+
+  test('HorizontalGallery should have 1 joining participant in the horizontal gallery', async ({ page, serverUrl }) => {
+    test.skip(isTestProfileStableFlavor());
+
+    const paul = defaultMockRemoteParticipant('Paul Bridges');
+    addVideoStream(paul, true);
+    const vasily = defaultMockRemoteParticipant('Vasily Podkolzin');
+    vasily.state = 'Connecting';
+
+    const participants = [paul, vasily];
+    const initialState = defaultMockCallAdapterState(participants);
+
+    await page.goto(buildUrlWithMockAdapter(serverUrl, initialState));
+
+    await waitForSelector(page, dataUiId(IDS.videoGallery));
+    expect(await stableScreenshot(page, { dismissTooltips: true })).toMatchSnapshot(
+      'horizontal-gallery-with-joining-participant.png'
+    );
+  });
+
+  test('HorizontalGallery should have multiple audio participants and 1 calling participant', async ({
```

Comment:
I'm confused by the terminology here: audio vs calling. What is a calling participant? Do you mean video participant?

Reply:
Oh, you mean PSTN participant. Please let's use the word PSTN uniformly for this. "Calling" is so overloaded in our context.

Reply:
I'd recommend making these PSTN participants more realistic. They should have the right `userId` type, a phone number for their `displayName` etc. I recommend adding a helper `defaultMockPSTNParticipant` and then setting the `state` to `Connecting` for all the tests here.

Reply:
So the tests here are to mimic for 1:N and PSTN, should I make a helper for both types?

Reply:
You can start with one. Add another only if the snapshots will differ.